src/System.Windows.Forms.Design/src/System/Windows/Forms/Design/DataGridViewCellStyleBuilder.cs

```diff
@@ -346,8 +349,9 @@ public DialogDataGridViewCellAccessibleObject(DataGridViewCell owner) : base(own
             {
             }
 
-            private string _name = "";
-            public override string Name
+            private string? _name = string.Empty;
+
+            public override string? Name
```

Comment:
nit: remove backing field?

Reply:
This is an override, so I assume that we should not remove this?

Reply:
not override but just the backing field.

```suggestion
            public override string? Name { get; set;}
```